5/run-client-local.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+export ADVERTISEDIP=`ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v -m1 '127.0.0.1'`
+export GATEWAYPORT=3001
+
+dotnet run --project ./Client/Client.csproj
```

Comment:
Yep. This is definitely not running in the background :)

Reply:
True, we'll have to kill that specific process in the stop if it's background :P

Reply:
Yep, but let's not do it. I do want clients to run in the background. Otherwise you won't see their console outputs and that defeats the whole purpose of having console clients.